tests/std/tests/P0896R4_ranges_alg_uninitialized_copy/test.cpp

```diff
@@ -215,19 +215,20 @@ struct memcpy_test {
         }
 
         { // Validate input shorter
-            int input[]  = {13, 55, 12345};
-            int output[] = {-1, -1, -1, -1};
+            int input[] = {13, 55, 12345};
+            int output[4];

```

Comment:
Ditto not a great idea. Note that the `construct_at` on 222 could hide a bug that results in the algorithm writing past the end of its proper output or have UB if `result.out` is past the end of `output`. (It's also somewhat silly to `construct_at` when we could simply `*result.out = -1;`.)